articles/introduction-to-extended-reality/index.md

```diff
@@ -0,0 +1,73 @@
+# Introduction to Extended Reality (XR)
+
+## What is Extended Reality (XR)?
+Extended reality (XR) is a term that encompasses real and virtual environments that are generated by wearable devices or computer technology to provide an immersive experience. It can also be described as a collection of all immersive technologies that combine real and virtual worlds. XR is growing in terms of advancement and application. This technology is still evolving because of ongoing development in human-machine interactions. 
+
+## Pillars of Extended Reality
+Extended reality encompasses three main pillars of immersive technology: Virtual Reality (VR), Augmented Reality (AR) and Mixed Reality (MR). 

```

Comment:
Augmented Reality (AR) and Mixed -> Augmented Reality (AR), and Mixed

Reply:
Done